docs/UserGuide.md

```diff
@@ -56,11 +56,11 @@ Format: `help`
 
 Adds a patient to the patient list.
 
-Format: `addPatient n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS d/TASK_DESCRIPTION…`
+Format: `addPatient [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG] [d/TASK_DESCRIPTION…]`
```

Comment:
- Would it be better to put `[t/TAG]` after `[d/TASK_DESCRIPTION…]` for consistency's sake, since it's the given order in the command summary list?
- `...` should be put outside the square brackets e.g. `[d/TASK_DESCRIPTION]...`
- `...` should be put after `[t/TAG]` e.g. `[t/TAG]...`  since it can be used multiple times (as mentioned in the original version). We could change these notations later on, but it might be better to stick to the existing conventions as of now.
- I think the square brackets should be removed for `n/NAME`, `p/PHONE_NUMBER`, `e/EMAIL` and `a/ADDRESS`, since they are compulsory fields in the original AB3. So the format would look like this:
```
addPatient n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [d/TASK_DESCRIPTION]… [t/TAG]…
```

Reply:
fixed the format